src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -178,11 +97,11 @@ private bool IsCsvFile(string filePath, IFormFile file)
             return _fileTypeService.HasMatchingMimeType(file, AllowedMimeTypesByFileType[ReleaseFileTypes.Data]) 
                    && _fileTypeService.HasMatchingEncodingType(file, CsvEncodingTypes);
         }
-        
-        private bool FileContainsSpacesOrSpecialChars(IFormFile file)
+
+        private bool FileContainsSpacesOrSpecialChars(string filename)
         {
-            return file.FileName.IndexOf(" ", Ordinal) > -1 || 
-                   file.FileName.IndexOfAny(Path.GetInvalidFileNameChars()) > -1;
+            return filename.IndexOf(" ", Ordinal) > -1 || 
+                   filename.IndexOfAny(Path.GetInvalidFileNameChars()) > -1;
```

Comment:
Don't think this validation is good enough. We ran into the problem just the other day that characters like `+` were allowed and this broke the 'Manage data' page completely for one of the releases.

Reply:
Add some extra val for ampersand - also on subject name/title which is what caused the initial problem.